core/templates/services/UpgradedServices.ts

```diff
@@ -543,6 +544,10 @@ export class UpgradedServices {
     upgradedServices['CollectionValidationService'] =
       new CollectionValidationService();
     upgradedServices['ComputeGraphService'] = new ComputeGraphService();
+    upgradedServices['ConnectionService'] = new ConnectionService(
+      new WindowRef(),
+      upgradedServices['HttpClient'],
+      new NgZone({}));
```

Comment:
Not required, `UpgradedServices.ts` is a stale file in the frontend. It is not used anywhere. So, no need to make changes to this file.

Reply:
Removed it.